swagger.json

```diff
@@ -1768,6 +1761,36 @@
                     }
                 }
             }
+        },
+        "/fleet/assets/{asset_id}/location": {
```

Comment:
This should be `locations` plural, because we're returning the list of locations for the time period. Should change the actual API & test too

Reply:
done